src/autogluon/bench/cloud/aws/deploy.sh

```diff
@@ -46,8 +46,8 @@ ECR_REPO=763104351884.dkr.ecr.us-east-1.amazonaws.com
 aws ecr get-login-password --region us-east-1 | docker login --username AWS --password-stdin $ECR_REPO
 
 echo "Running CDK deploy"
-cdk deploy --app $CDK_PATH $STATIC_RESOURCE_STACK_NAME
-cdk deploy --app $CDK_PATH $BATCH_STACK_NAME
+cdk deploy --app $CDK_PATH $STATIC_RESOURCE_STACK_NAME --require-approval never
+cdk deploy --app $CDK_PATH $BATCH_STACK_NAME --require-approval never
```

Comment:
Are there any security or unexpected side-effects that we should be concerned about when skipping the approval?

Reply:
So this is pretty self contained since it takes in arguments of specific stack names. The intention of bypassing the approval is because we want to soon integrate `autogluon.bench` with `autogluon` CI process, and we would want to auto proceed with any prompt.